src/lib/units/month.js

```diff
@@ -12,6 +12,7 @@ import isArray from '../utils/is-array';
 import indexOf from '../utils/index-of';
 import { createUTC } from '../create/utc';
 import getParsingFlags from '../create/parsing-flags';
+import isUndefined from '../utils/is-undefined';
 
```

Comment:
Ah, my bad. I forgot to remove that import. Will do that.
So should I use `typeof m === 'undefined'` or `!m`?